ast/expressions_test.go

```diff
@@ -101,3 +104,31 @@ func (tc *testExpressionsSuite) TestExpresionsVisitorCover(c *C) {
 		v.node.Accept(visitor1{})
 	}
 }
+func (tc *testExpressionsSuite) TestExpresionsRestore(c *C) {
+	parser := parser.New()
+	testNodes := []struct {
+		sourceSQL string
+		expectSQL string
+	}{{"select ++1", "SELECT ++1"},
+		{"select --1", "SELECT --1"},
+		{"select -+1", "SELECT -+1"},
+		{"select -1", "SELECT -1"}}
+
+	for _, node := range testNodes {
+		stmt, err := parser.ParseOneStmt(node.sourceSQL, "", "")
+		comment := Commentf("source %#v", node)
+		c.Assert(err, IsNil, comment)
```

Comment:
It seems that we do not need line120 ~ line127.
We can define restoreSql as stmt.Restore().

Reply:
i'm sorry,i don't understand you

Reply:
I mean we do not need to make up `restoreSql` manually
We replace line129~136 to
``` go
restoreSql := stmt.Restore()
```